recipe/meta.yaml

```diff
@@ -26,37 +26,37 @@ requirements:
   run:
     - python >=3.8,<3.12
     - aiobotocore >=2.4,<3
-    - zarr >=2.14,<=2.16
-    - numpy >=1.20,<2
+    - aioitertools >=0.11,<1
+    - botocore >=1.31,<2
+    - click >=8.1,<9
     - donfig >=0.7,<1.0
-    - typing-extensions >=4.7.0,<5.0.0
-    - pydantic >=1.10,<2
-    - httpx >=0.23,<1.0
-    - cachetools >=5.3,<6
-    - structlog >=23.1,<24
     - email_validator >=2,<3
-    - click >=8.1,<9
-    - rich >=12.6,<14.0
-    - typer >=0.6,<1.0
+    - httpx >=0.23,<1.0
+    - numpy >=1.20,<2
+    - pydantic >=1.10,<2
+    - python-dateutil >=2,<3
     - pyyaml >=6,<7.0
+    - rich >=12.6,<14.0
     - ruamel.yaml >=0.17,<1.0
-    - python-dateutil >=2,<3
+    - structlog >=23.1,<24
+    - typer >=0.6,<1.0
     - uvloop >=0.17,<1  # [unix]
+    - zarr >=2.14,<=2.16
     - __unix  # [unix]
     - __win   # [win]
-    - aioitertools >=0.11,<1
-    - xarray >=v2022.12.0
+    # - cachetools
+    # - typing-extensions
```

Comment:
this are not actually required but I expect `pip check` to complain because our package lists them as required. I'll revert after confirming and we can fix in our next release